addons/base_automation/static/tests/base_automation_error_dialog.js

```diff
@@ -70,7 +70,7 @@ QUnit.module("base_automation", {}, function () {
         const dialogOpened = new Promise((resolve) => {
             env.services.dialog.bus.on("ADD", null, async ({ class: dialogClass, props }) => {
                 assert.equal(props.data.context, errorContext, "Received the correct error context");
-                assert.equal(dialogClass, BaseAutomationErrorDialog, "Should open the correct dialog class");
+                assert.equal(dialogClass.__proto__, BaseAutomationErrorDialog, "Should open the correct dialog class");
```

Comment:
that's a shame. Couldn't we check the content of the dialog instead?

Reply:
instanceof